bundler/spec/runtime/platform_spec.rb

```diff
@@ -365,7 +365,7 @@
     simulate_windows do
       install_gemfile <<-G
         source "#{file_uri_for(gem_repo1)}"
-        gem "nokogiri", :platforms => [:mingw, :mswin, :x64_mingw, :jruby]
+        gem "nokogiri", :platforms => [:windows, :mswin, :mswin64, :mingw, :x64_mingw, :jruby]
```

Comment:
What's this change for?

Reply:
Same reason as https://github.com/rubygems/rubygems/pull/6391#discussion_r1120877604